### PR TITLE
[CF-378] Fix `CheckNetwork` action's output

### PR DESCRIPTION
**Pull request by miarmak**
Change 'some messy completely not understandable output' to 'something
human readable'.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/actions/check_networks.py [hunk 1]
```diff
@@ -143,7 +145,7 @@ def run(self, **kwargs):
 class ComputeInfo(object):
     def __init__(self, src_compute, search_opts):
         search_opts['all_tenants'] = True
-        self.instances = {s.id for s in src_compute.get_instances_list(
+        self.instances = {str(s.id) for s in src_compute.get_instances_list(
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/check_networks.py]: as far as i know the `id` of an instance is always a string.

File: cloudferrylib/os/actions/check_networks.py [hunk 2]
```diff
@@ -293,8 +295,8 @@ def get_overlapping_external_subnets(self, dst_info, ext_net_map):
                               src_subnet['id'], src_subnet['network_id'])
                     continue
 
-                overlap = {'src_subnet': src_subnet['id'],
-                           'dst_subnet': dst_subnet['id']}
+                overlap = {'src_subnet': str(src_subnet['id']),
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/check_networks.py]: same here, `id` ids a string

File: cloudferrylib/os/actions/check_networks.py [hunk 3]
```diff
@@ -293,8 +295,8 @@ def get_overlapping_external_subnets(self, dst_info, ext_net_map):
...
+                           'dst_subnet': str(dst_subnet['id'])}
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/check_networks.py]: ditto

File: cloudferrylib/os/actions/check_networks.py [hunk 4]
```diff
@@ -332,7 +334,7 @@ def get_overlapping_networks(self, dst_info):
                 try:
                     network.check_network_overlapping(dst_net)
                 except exception.AbortMigrationError:
-                    nets_with_overlapping_subnets.append(network.id)
+                    nets_with_overlapping_subnets.append(str(network.id))
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/check_networks.py]: ditto

File: cloudferrylib/os/actions/check_networks.py (hunk 1 above)
Comment (miarmak) [inline cloudferrylib/os/actions/check_networks.py]: Nope, it's `unicode`.

File: cloudferrylib/os/actions/check_networks.py (hunk 2 above)
Comment (miarmak) [inline cloudferrylib/os/actions/check_networks.py]: The same as above.

File: cloudferrylib/os/actions/check_networks.py (hunk 3 above)
Comment (miarmak) [inline cloudferrylib/os/actions/check_networks.py]: The same as above.

File: cloudferrylib/os/actions/check_networks.py (hunk 4 above)
Comment (miarmak) [inline cloudferrylib/os/actions/check_networks.py]: The same as above.

File: cloudferrylib/os/actions/check_networks.py (hunk 1 above)
Comment (SVilgelm) [inline cloudferrylib/os/actions/check_networks.py]: Ok. python cannot compare the unicode and simple strings???

**Comment by miarmak** [comment]
Should be implemented in other way. Closing for now.

**Pull request closed without merging**